### ISSUE TITLE
Add Bandcamp.com

### ISSUE THREAD

**Issue by jgillman** [borismus/keysocket#128]
Bandcamp.com support would be great! I've been reading through their JS for the last hour or so but I haven't been able to tap into it. The most reliable method seems to be just simulating clicks on the play/pause and next/previous elements on the page:

``` javascript
// Click play / pause
document.querySelector('.playbutton').click()

// Click next / previous
document.querySelector('.nextbutton').click()
document.querySelector('. prevbutton').click()
```

This works fine on album pages (e.g. https://cascaderecords.bandcamp.com/album/1001-nights-ep) but doesn't work well on any other page as `.playbutton` either doesn't exist or can match more than one element.

**Comment by feedbee** [comment]
Would you like to make pull request for this feature?

**Comment by vinyldarkscratch** [comment]
Even though this won't work with custom URLs for Bandcamp sites (a shame, but at least with how the extension is currently written, it's not possible to detect such sites), it looks like this service has already been added.  Just updated the readme with #208!

**Comment by feedbee** [comment]
Released in 0.8.1.
